<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>554</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MailMe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>581</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="1" column="1">
     <widget class="QLabel" name="labelExpediteur">
      <property name="text">
       <string>Expediteur : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineExpediteur">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelSujet">
      <property name="text">
       <string>Sujet :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineSujet"/>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QTextEdit" name="textMessage"/>
    </item>
    <item row="6" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelMessage">
        <property name="text">
         <string>Message :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelDestinataire">
      <property name="text">
       <string>Destinataire : </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineDestinataire"/>
    </item>
    <item row="8" column="2">
     <layout class="QHBoxLayout" name="hlEnvoyer">
      <item>
       <widget class="QPushButton" name="bouttonEnvoyer">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="labelPiecesJointes">
      <property name="text">
       <string>Pieces-jointes :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="linePiecesJointes"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelServeur">
      <property name="text">
       <string>Serveur Smtp :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineServeur"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineServeur</tabstop>
  <tabstop>lineExpediteur</tabstop>
  <tabstop>lineDestinataire</tabstop>
  <tabstop>lineSujet</tabstop>
  <tabstop>textMessage</tabstop>
  <tabstop>linePiecesJointes</tabstop>
  <tabstop>bouttonEnvoyer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
